docs/oathkeeper/grpc-middleware.md

```diff
@@ -0,0 +1,43 @@
+---
+id: grpc-middleware
+title: gRPC middleware
+---
+
+In addition to exposing Ory Oathkeeper as a HTTP proxy, you can directly embed it into your Go application as a gRPC middleware.
+The gRPC traffic will then be handled by the middleware as if it were a standalone Ory Oathkeeper instance, just without the added
+network hop.
+
```

Comment:
Please add a warning that this feature is experimental and we give no API compat guarantees

```
:::warning

the message

:::
```

Reply:
done